utils/build_swift/postbuild_steps.py

```diff
@@ -0,0 +1,115 @@
+#===--- postbuild.py -----------------------------------------------------===#
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2018 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https:#swift.org/LICENSE.txt for license information
+# See https:#swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+#===----------------------------------------------------------------------===#
+
+import os
+import platform
+import shutil
+
+import swift_build_support.swift_build_support.products as products
+import swift_build_support.swift_build_support.tar as tar
+import swift_build_support.swift_build_support.shell as shell
+from swift_build_support.swift_build_support.targets \
+    import StdlibDeploymentTarget
+
```

Comment:
Please add a second new-line between top-level declarations to conform with PEP8 styling guidelines. It's also standard practice in modules that "export" top-level symbols to define the magic `__all__` list. For this module we probably just need:

```Python
__all__ = [
    'execute',
]
```